content/doc/tutorials/build-a-labview-app.adoc

```diff
@@ -0,0 +1,384 @@
+
+. Next, we will modify the VIs. First, open `Add.vi`. This is a simple VI that adds two numbers together and returns a result.
+[.boxshadow]
+image:tutorials/labview-28-add-vi-orig.png[alt="add.vi orig",width=100%]
+ 
+. Make some cosmetic changes to `Add.vi` (don’t change anything that would alter the functionality of the VI), like so:
+[.boxshadow]
+image:tutorials/labview-29-add-vi-changed.png[alt="add.vi changed",width=100%] 
+
+. Click *File >> Save* to save the VI changes, and close `Add.vi`. 
+
+. Now, open `Subtract.vi`. This is a simple VI that subtracts two numbers and returns a result.
+[.boxshadow]
+image:tutorials/labview-30-subtract-vi-orig.png[alt="subtract.vi orig",width=100%]
+
+. Make cosmetic changes to `Subtract.vi`; for example:
+[.boxshadow]
+image:tutorials/labview-31-subtract-vi-changed.png[alt="subtract.vi changed",width=100%]
+ 
+. Click *File >> Save* to save changes, and close `Subtract.vi`.
+
+. Close `Jenkins.lvproj`. Save files if prompted.
+
+. With the Git terminal open, make sure we are still in the `../myApplication` directory. Run the following commands to push our changes to the _myBranch_ branch on the _myApplication_ repository:
+
+    git add .
+    git commit -m “Cosmetic VI changes”
+    git push origin myBranch
+ 
+. With our changes pushed to the Branch, we’ll go on GitHub to create a Pull Request. First, navigate to the _myApplication_ repository on https://www.github.com[GitHub].
+
+. Navigate to the ‘myBranch’ Branch using the dropdown:
+[.boxshadow]
+image:tutorials/labview-32-navigate-to-github-branch.png[alt="navigate to github branch",width=100%]
+ 
+. On the _myBranch_ page, click on *New pull request*.
+
+. Fill out the pull request details as shown. Click *Create pull request* to create the request.
+[.boxshadow]
+image:tutorials/labview-33-open-pr-details.png[alt="open PR details",width=100%]
+
+. Wait for Jenkins to perform its check based on the interval you set, or navigate to your *Jenkins dashboard >> GitHub Builder (or whatever you named your Jenkins pipeline) >> Scan Organization Now*. Then, navigate to *myApplication >> Scan Repository Now*.
+
+. To view the progress of your job, you can navigate to the *Pull Requests* tab.
+[.boxshadow]
+image:tutorials/labview-34-jenkins-pr-tab.png[alt="jenkins pr tab",width=100%] 
+ 
+. Click on the pull request name (_PR-1_ in this case). The progress of each step is displayed on the main page. The _Build History_ section on the bottom left displays the jobs run. 
+[.boxshadow]
+image:tutorials/labview-35-jenkins-pr-detail.png[alt="jenkins pr detail",width=100%]
+
+. If the job was successful, each stage should be Green, and the latest Build in the _Build History_ section should have a blue dot next to it. Additionally, you’ll see the latest artifacts (files you can upload to Jenkins), and the unit test results:
```

Comment:
Perhaps my understanding of this process is misconstrued, but I am archiving artifacts during my postClean step with this command:

`archiveArtifacts artifacts: 'TEMPDIR/**'`

The impression I got from [this page](https://jenkins.io/doc/pipeline/tour/tests-and-artifacts/) was that this archival could be useful. If this is not a great use case, please let me know!

Reply:
The use-case is good, it means artifacts are copied from `TEMPDIR` to `path-to-job/archive` so that Jenkins users can download them later. From user's perspective it may make more sense to mention downloading.